form.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+require 'functions.php';
+
+$db = new PDO("mysql:host=db; dbname=collection_app", "root", "password");
+$db ->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+if( (!empty($_POST['title'])) && (!empty($_POST['author'])) && (!empty($_POST['released'])) && (!empty($_POST['genre'])) && (!empty($_POST['page_count'])) ) {
+    insertToDB($db, $_POST['title'], $_POST['author'], $_POST['genre'], $_POST['page_count'], $_POST['released']);
```

Comment:
Some basic sanitisation might be nice here for these inputs.

Using simple functions like trim() (to remove leading/trailing whitespace) and htmlspecialchars() (to remove any dodgy HTML) would be a good idea! 

Remember reusable sanitization functions are a wonderful thing! 😆